pkg/pod/pod.go

```diff
@@ -595,3 +576,20 @@ func usesWindows(tr *v1beta1.TaskRun) bool {
 	osSelector := tr.Spec.PodTemplate.NodeSelector[osSelectorLabel]
 	return osSelector == "windows"
 }
+
+// checkRequestedVolumeMounts checks the existing volume mounts in the current Container and returns a slice
+// of VolumeMounts to be added to the container.
+func checkRequestedVolumeMounts(c corev1.Container, requestedVolumeMounts []corev1.VolumeMount) []corev1.VolumeMount {
```

Comment:
is it right that this function is getting missing volume mounts? it'd be useful to use a precise name for what this function is checking in the requested volumes

```suggestion
// checkRequestedVolumeMounts checks the existing volume mounts in the current Container and returns a slice
// of VolumeMounts to be added to the container.
func getMissingVolumeMounts(c corev1.Container, requestedVolumeMounts []corev1.VolumeMount) []corev1.VolumeMount {
```